lambdas/TranscodeTaskOutputLambdaFunction/index.js

```diff
@@ -0,0 +1,17 @@
+// Because the result of a Fargate task is not sufficient for sending a proper
+// callback, this function takes the entire task input and builds a better
+// result that gets passed to the callback task.
+
+exports.handler = async (event) => {
+  console.log(JSON.stringify({ msg: 'State input', input: event }));
+
+    const result = {
```

Comment:
So there's _no_ output at all from the Fargate task?  Or you just don't need any of it for this result?

I guess you already know it succeeded, if this is getting called.  And you know what Destination was requested.

Reply:
No value gets returned from Fargate to SF. The ECS task may try to return something, but it doesn't make it back to the state machine. The Fargate-calling state *does* return something, but it just returns the input that was provided to it. So yes, basically this is just making the assumption that if it's happening, the task succeeded and we can use those input values as output values.